brave/brave-core
# Fixes tap image not showing every time

Resolves https://github.com/brave/brave-browser/issues/8964

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- go to rewards page
- make sure that tap asset is always shown

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
